doc/software.rst

```diff
@@ -0,0 +1,75 @@
+Software
+========
+
+Install the latest version of the library directly from
+`PyPI <https://pypi.python.org/pypi?:action=display&name=luma.oled>`__
+with::
+
+  $ sudo -H pip install --upgrade luma.oled
+
+This will normally retrieve all of the dependencies ``luma.oled`` requires and
+install them automatically.
+
+Installing Dependencies
+-----------------------
+If ``pip`` is unable to automatically install its dependencies you will have to
+add them manually.
+
+``luma.oled`` relies on the following packages.
+
+.. csv-table::
+  :header: "Package", "Description"
+  :widths: 15, 25
+
```

Comment:
see my prev luma.lcd comment about listing the dependencies.

Reply:
I've removed the references and reworded the section both here and in ``luma.lcd``